util/eventbus/bus.go

```diff
@@ -13,12 +13,29 @@ import (
 	"tailscale.com/util/set"
 )
 
+type publishedEvent struct {
+	Event     any
+	From      *Client
+	Published time.Time
```

Comment:
```suggestion
	Published time.Time  // When the event was enqueued in the global order
```

Reply:
But if that's correct, then I'm not sure what "Accepted" means, do I have it inverted?